sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
> In that case, I would add a regression support and add support for array-like.

I mean. I would add a test for this. By the way, we should update the documentation.